documentation/API_documentation/QoD_Stable_Latency_API.md

```diff
@@ -0,0 +1,181 @@
+
+* Latency QoS session with 1H duration and QoS-profile "LOW\_Latency" mapping,
+* App-Flow is specified for UDP protocol with UE-Terminal IP address (ueAddr=10.0.0.1), Application server network (asAddr=54.204.25.0/28) and Port number (asPorts=33001).
+
+Please note, the credentials for API authentication purposes need to be adjusted based on target security system configuration.
+
+| Snippet 1. Create QoS session to manage latency |
+| ----------------------------------------------- |
+| curl -X 'POST' '[https://\[replace](https://%5Breplace) with your base URL]/qos-senf/v1/sessions'   <br>    -H 'accept: application/json' <br>    -H 'Content-Type: application/json'<br>    -H "Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbG...."<br>    -d '{<br>     "duration": 3600,<br>     "ueAddr": "10.0.0.1",<br>     "asAddr": "54.204.25.0/28",<br>     "asPorts": "33001",<br>     "protocolOut": "UDP",<br>     "qos": "LOW\_Latency",<br>     "notificationUri": "[https://your-callback-server.com/notifications](https://your-callback-server.com/notifications)",<br>     "notificationAuthToken": "c8974e592c2fa383d4a3960714"<br>   }' |
+<br>
+Snippet 2, elaborates sample QoS notification "SESSION\_TERMINATION" message distributed from QoD backend to client callback function.
+
+| Snippet 2. Sample QoS session notification |
+| ------------------------------------------ |
+| {<br>   "sessionId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",   "event": "SESSION\_TERMINATED"<br>} |
+
+### 4.6 FAQ's
+
+* Q: *My application server components are distributed between multiple public IP addresses. In this case how the asAddr parameter should be defined?*
+A: You can use the network segment for asAddr. Currently there are no restrictions to the size of the network segment.
+You can also use the whole IPv4 address range by setting 0.0.0.0/0. Please note
+* Q: *I want to set QoS profile for communication between two UE pieces. Is this possible?*
+A: Communication with UE hosts from the core network is controlled by the selected QoS profile in both ways. You will need to create two QoS sessions.
+Both sessions should set corresponding UE IPv4 address as AS parameter.
+* Q: How is the prioritized connection (App-Flow) actually managed and described?
+For example, the connectivity (App-Flow) between UE IP and AS IP has a requirement for prioritization - how to configure it?
+<span class="colour" style="color:var(--vscode-unotes-wsyText)">A: Using the API please create QoS session with reservation time-slot, provide IP/Port and Protocol to identify App-Flow and quality is managed automatically.  </span>
+* Q: What is the purpose of storing QoS session ID after successful create operation?
+Is it important to store it due to subsequent PATCH or DELETE calls, or something else?
+<span class="colour" style="color:var(--vscode-unotes-wsyText)">A: just for subsequent calls or it will be sent in the notification as well, and PATCH is not supported.</span>
```

Comment:
It's propbably important for reconciliation, i.e. developer says 'QoS session abc123 did not deliver the quality I ordered, please check'

Reply:
Taken FAQs out of current PR, rework needed.

Reply:
Agreed